app/forms/idv/api_image_upload_form.rb

```diff
@@ -353,6 +353,10 @@ def update_analytics(client_response:, vendor_request_time_in_ms:)
       birth_year = client_response.pii_from_doc&.dob&.to_date&.year
       zip_code = client_response.pii_from_doc&.zipcode&.to_s&.strip&.slice(0, 5)
       issue_year = client_response.pii_from_doc&.state_id_issued&.to_date&.year
+      captured_result = document_capture_session&.load_result
+      processed_selfie_count = (selfie_image_fingerprint ? 1 : 0)
+      past_selfie_count = (captured_result&.failed_selfie_image_fingerprints || []).length
+      selfie_attempts = past_selfie_count + processed_selfie_count
```

Comment:
can we put this in its own private method?

Reply:
@amirbey should be resolved now

Reply:
it looks like we're still logging selfie attempts for instances where a liveness check is not required 🤔